### DISCUSSION TITLE
Frame Capture Not Working

### DISCUSSION THREAD

**Discussion by Agonza-Netta** [mifi/lossless-cut#2222] [Q&A]
Hello,
The **frame capture feature** in my installation of losslesscut seems to be **deactivated and not working at all**. Considering I have the latest versions of lossletcut and FFmpeg installed, I am unsure of the reason for this issue. It is not activated when I try to use it with .MP4 and .MOV files. Where should I start to look for...

**Comment by mifi** [comment]
Hi. What do you mean by that exactly?

**Comment by Agonza-Netta** [comment]
I am trying to save frames of videos with LosslessCut (take snapshots/capture frames of the loaded video). 

However, this function does not work at all for me. When I click on the camera icon with the mouse, nothing happens, even when I try to take a picture with the C shortcut, nothing happens.

This function is completely disabled in my installation, what can I do to reactivate it?

**Reply by mifi** [reply]
Oh. can you tell me more about your computer and version etc? see questions from here: https://github.com/mifi/lossless-cut/issues/new?assignees=&labels=&projects=&template=2-bug.yml

You can also try to toggle the Settting "Snapshot capture method" and see if it makes a difference

**Comment by OscarBarkTaigatech** [comment]
Hello!

I was recently pulling my hair out trying to debug this issue until I realized that I had at some point in the past set a custom "Working Directory" under "Options affecting exported files" in the global settings.

Clearing that setting solved it to me, so totally my fault!

Perhaps in the future, LosslessCut could make it a bit clearer where snapshots are exported to when you take the snapshot, e.g. by a brief toast message? (Or is that another option I've disabled in the past perhaps?)

_Note:_ I can see the "export in progress screen" (the rainbow fire hydrant) flash very briefly when snapshotting, but too briefly to be able to parse any information from it.

**Reply by mifi** [reply]
hi! when you capture a snapshot, you get a notification "Success! Screenshot captured to: /path/to/file-00.00.07.277.jpg"

maybe you turned off all notifications under settings?
